Dockerfile-test

```diff
@@ -0,0 +1,18 @@
+FROM continuumio/miniconda3:4.3.11
```

Comment:
You probably don't want to pin this to a version

sorry for the drive by commenting. talk just started.